packages/playground/src/app/simple-page/simple-page.component.ts

```diff
@@ -0,0 +1,41 @@
+import { CommonModule } from '@angular/common';
+import { HttpClient } from '@angular/common/http';
+import { Component, inject } from '@angular/core';
+import { QueryProvider } from '@ngneat/ng-query';
+import { SubscribeModule } from '@ngneat/subscribe';
+
+interface Repository {
+  name: string;
+  description: string;
+  subscribers_count: number;
+  stargazers_count: number;
+  forks_count: number;
+}
+
+@Component({
+  standalone: true,
+  imports: [CommonModule, SubscribeModule],
+  template: `
+    <ng-container *subscribe="repo$ as repo">
+      <p *ngIf="repo.isLoading">Loading...</p>
+      <p *ngIf="repo.error">An error has occurred: {{ repo.error }}</p>
+
+      <div *ngIf="repo.isSuccess">
+        <h1>{{repo.data.name }}</h1>
+        <p>{{repo.data.description }}</p>
+        <strong>👀 {{repo.data.subscribers_count }}</strong>
+        <strong>✨ {{repo.data.stargazers_count }}</strong>
+        <strong>🍴 {{repo.data.forks_count }}</strong>
+        <div>{{repo.isFetching ? "Updating..." :  ""}}</div>
+      </div>
+    </ng-container>
+  `,
+})
+export class SimplePageComponent {
+  useQuery = inject(QueryProvider);
```

Comment:
@NetanelBasal Is this OK to `inject` within the component or do you prefer to move it into a service?

Reply:
Our examples should show the best practices, so let's always user services.

Reply:
Sure, I'll update the code.